packages/www/docs/api-reference/ingest.mdx

```diff
@@ -31,8 +31,8 @@ playback URL pairs.
 [
   {
     ingest: "rtmp://<reg>-rtmp.livepeer.com/live",
-    playback: "https://<reg>-cdn.livepeer.com/hls",
-    base:"https://<reg>-cdn.livepeer.com/"
+    playback: "https://<reg>-livepeercdn.com/hls",
+    base:"https://<reg>-livepeercdn.com/"
```

Comment:
I don't think we support these domains since we only have `livepeercdn.com`

Maybe keep these regional ones as `*.livepeer.com`, or maybe even just remove this part of the doc as Eli suggested.